README.md

```diff
@@ -1,8 +1,20 @@
 # multihash
 
+> hashpipe - pipe iff the hash matches
+
 Multihash is a protocol for differentiating outputs from various well-established cryptographic hash functions, addressing size + encoding considerations.
 
-It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See https://github.com/jbenet/random-ideas/issues/1 for a longer discussion.
+It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See [this GitHub thread](https://github.com/jbenet/random-ideas/issues/1) for a longer discussion.
+
+## Table of Contents
+
+- [Example](#example)
+- [Format](#format)
+- [Implementations:](#implementations)
+- [Table for Multihash v1.0.0-RC (semver)](#table-for-multihash-v100-rc-semver)
+- [Disclaimers](#disclaimers)
+- [Contribute](#contribute)
+- [License](#license)
```

Comment:
Do we need a table of contents? the Readme is pretty short. i'm worried it will be me more trouble to maintain working + updated